_config.ts

```diff
@@ -210,13 +176,35 @@ site.ignore(
   // "subhosting",
 );
 
-site.scopedUpdates((path) => path == "/overrides.css");
-site.use(
-  checkUrls({
-    external: false, // Set to true to check external links
-    output: "_broken_links.json",
-    ignore: ["https://www.googletagmanager.com"],
-  }),
+site.scopedUpdates(
+  (path) => path == "/overrides.css",
+  (path) => /\.(js|ts)$/.test(path),
+  (path) => path.startsWith("/api/deno/"),
 );
 
+const SKIP_CHECK_URLS = (Deno.env.get("SKIP_CHECK_URLS") || "false")
+  .toLowerCase();
+
+if (SKIP_CHECK_URLS !== "true") {
+  log.info(`${cliNow()} Enabling broken link checker`);
```

Comment:
Nit: wording sounds like the link checker itself is broken

Reply:
added a dash, so now it is "Enabling broken-link checker"